spec/game_runner_spec.rb

```diff
@@ -3,24 +3,48 @@
 
 RSpec.describe GameRunner do
   describe "#run" do
-    it "on start up, an empty grid is rendered on the screen with numbers at the top of each column and letters next to each row" do
-      grid_template = <<~GRID
-        1  2  3
-       __ __ __
-    A |  |  |  |
-      |__|__|__|
-    B |  |  |  |
-      |__|__|__|
-    C |  |  |  |
-      |__|__|__|
-      GRID
-
-      stdout = StringIO.new
-      game_start = GameRunner.new(stdout)
-
-      game_start.run
-
-      expect(stdout.string).to eq(grid_template)
+    context "on start up" do
+      it "renders an empty grid with numbers at the top of each column and letters next to each row" do
+        grid_template = <<~GRID
+          1  2  3
+         __ __ __
+      A |  |  |  |
+        |__|__|__|
+      B |  |  |  |
+        |__|__|__|
+      C |  |  |  |
+        |__|__|__|
+        GRID
+
+        stdout = StringIO.new
+        stdin = StringIO.new
+        game_start = GameRunner.new(stdout, stdin)
+
+        game_start.run
+
+        expect(stdout.string).to include(grid_template)
+      end
+
+      it "prompts a user for input" do
+        stdout = StringIO.new
+        stdin = StringIO.new
+        game = GameRunner.new(stdout, stdin)
+
+        game.run
+
+        expect(stdout.string).to include("Enter your move >")
+      end
+
+      it "waits for user input" do
+        stdout = StringIO.new
+        stdin = StringIO.new
+        game = GameRunner.new(stdout, stdin)
+        allow(stdin).to receive(:gets).and_return("No validations yet")
```

Comment:
I think it would be more instructive to enter a bad coordinate. Also, we know this doesn't take into account how to end the game, since it will keep waiting for input infinitely.